aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -582,6 +582,18 @@ module std::features {
         is_enabled(ABORT_IF_MULTISIG_PAYLOAD_MISMATCH)
     }
 
+    /// Whether the multisig v2 fix is enabled. Once enabled, the multisig transaction execution will explicitly
+    /// abort if the provided payload does not match the payload stored on-chain.
```

Comment:
This also affects other stuff such as the authentication key and gas payer right?

Reply:
Updated!